x-pack/plugins/enterprise_search/public/applications/app_search/routes.ts

```diff
@@ -39,7 +39,7 @@ export const ENGINE_REINDEX_JOB_PATH = `${ENGINE_SCHEMA_PATH}/reindex_job/:reind
 export const ENGINE_CRAWLER_PATH = `${ENGINE_PATH}/crawler`;
 export const ENGINE_CRAWLER_DOMAIN_PATH = `${ENGINE_CRAWLER_PATH}/domains/:domainId`;
 
-export const META_ENGINE_CREATION_PATH = '/meta_engine_creation';
+export const META_ENGINE_CREATION_PATH = `${ENGINES_PATH}/new_meta_engine`; // This is safe from conflicting with an :engineName path because engine names cannot have underscores
```

Comment:
[AS] This is not the same route as the old standalone UI - I think it was `/meta_engines/new` there.

I'm not 100% sold on `/engines/new_meta_engine` myself - do folks prefer something more like `/engines/new/meta_engine`?